QATCH/common/licenseManager.py

```
@@ -0,0 +1,865 @@
+
+        # cursor.execute("SELECT * FROM subscribers")
+        # print(cursor.fetchall())
+        # cursor.execute("SELECT * FROM licenses")
+        # print(cursor.fetchall())
+
+    def close(self):
+        try:
+            if self.connection.open:
+                self.connection.commit()
+                self.connection.close()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)